.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ jobs:
         run: |
           npm install
           npm install @web/test-runner-playwright --no-save
+          npx playwright install
```

Comment:
It should actually be using `npx playwright install --with-deps`. There was a post about this in #web-platform a few weeks ago. It seems like [main](https://github.com/BrightspaceHypermediaComponents/siren-sdk/blob/main/.github/workflows/ci.yml#L16C11-L16C45) is already updated with this change so you should just need to pull in master

Reply:
Updated

Reply:
Because this is pointing at another branch, the changes that were pulled are all showing up. I think if you merge main into Kevin's branch it'll fix this